components/tyk-mdcb/values.yaml

```diff
@@ -183,13 +183,16 @@ mdcb:
   trackAllPaths: false
 
   # If enabled, aggregate data will be generated per minute.
-  storeAnalyticsPerMinute: false   
-
+  storeAnalyticsPerMinute: false
+  
+  security:
+    secret: 1234567890
+    enableHttpSecureEndpoints: false
   probes:
     # This port lets MDCB allow standard health checks.
     # It also defines the path for liveness and readiness probes.
-    # It is used to set TYK_MDCB_HEALTHCHECKPORT
-    healthCheckPort: 8181
+    # It is used to set TYK_MDCB_HTTPPORT
+    httpPort: 8181
```

Comment:
Renaming field will be a breaking change. Is this change a must?

Reply:
Yes. That will be better.
We can mark `healthCheckPort` as deprecated but still support it. `httpPort` will take the precedence if both are set.

Reply:
done!